sorts/count_sort.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import 	"fmt"
+		
+
+func main() {
+	i:=0
+	fmt.Print("Enter Number of Elements: ")
+  	var n int
+  	fmt.Scan(&n)
+  	arr:=make([]int, n)
+  	for(i<n){
+  	fmt.Print("Enter Element ",i+1,": ")
+    fmt.Scan(&arr[i])
+    i=i+1
+  }
+  	var max int 					//finds the maximum from an array
+ 	max = arr[0]
+ 	i=0
+ 	for(i<n){
```

Comment:
Please re-format the code with `golint`. I see that all your loop can follow the format
```{go}
for i := initialValue; i < condition; i++ {
  ... some part of code
}
```
Please refactor all your code as the template above.